onnxruntime/core/providers/nnapi/nnapi_builtin/builders/shaper.cc

```diff
@@ -6,67 +7,65 @@
 using std::string;
 using std::vector;
 
+std::pair<uint32_t, uint32_t> ComputeConvOutputShape(const uint32_t input_size_y, const uint32_t input_size_x,
+                                                     const uint32_t weight_size_y, const uint32_t weight_size_x,
+                                                     const vector<int32_t>& onnx_pads,
+                                                     const vector<int32_t>& onnx_strides,
+                                                     const vector<int32_t>& onnx_dilations) {
+  int32_t padding_top = onnx_pads[0];
+  int32_t padding_bottom = onnx_pads[2];
+  int32_t padding_left = onnx_pads[1];
+  int32_t padding_right = onnx_pads[3];
+  int32_t stride_y = onnx_strides[0];
```

Comment:
If the height padding 'before' and 'after' values are in 0 and 2, wouldn't bottom be 0 and top 2?